### PR TITLE
Change Task from CDex conformant to PAS

### PR DESCRIPTION
**Pull request by elsaperelli**
# Summary
This PR changes the Task resource in the `dtr_bundle_patient_pat015.json` file from a CDex Task to a PAS Task. This is to reflect changes to the DTR Light EHR Test suite in the davinci-dtr-test-kit based on this PR comment: https://github.com/inferno-framework/davinci-dtr-test-kit/pull/35#discussion_r1868310626.

## New behavior
- Task is now conformant to the [PAS Task](http://hl7.org/fhir/us/davinci-pas/StructureDefinition/profile-task) profile rather than the [CDex Task Profile](http://hl7.org/fhir/us/davinci-cdex/StructureDefinition/cdex-task-attachment-request).

## Code changes
- Example PAS Task from the [PAS IG](http://hl7.org/fhir/us/davinci-pas/STU2/Task-AdditionalInformationTaskExample.html) instead of CDex Task.

## Testing guidance
- Run this branch locally and run [this branch](https://github.com/inferno-framework/davinci-dtr-test-kit/pull/35) of the `davinci-dtr-test-kit` and run the DTR Light EHR Profiles Tests in the DTR Light EHR Test Suite using the Inferno Reference Server (Local) preset.

### CODE REVIEW

File: resources/dtr_bundle_patient_pat015.json [hunk 1]
```diff
@@ -2721,7 +2737,123 @@
...
       "request": {
         "method": "PUT",
-        "url": "Task/cdex-task-example19"
+        "url": "Claim/MedicalServicesAuthorizationExample"
+      }
+    },
+    {
+      "resource": {
+        "resourceType": "Coverage",
+        "id": "InsuranceExample",
```
Comment (karlnaden) [inline resources/dtr_bundle_patient_pat015.json:2746]: Still trying to figure out if there is a id style we should use, but I think for sure we should not have "Example" in the id. The rest of this file has a general approach to ids that I think we should probably follow:

- for patient-linked instances: <3 letter resource type abbreviation><patient #>[-<something to distinguish multiple instances of the same type>]
- for shared instances like organizations <3 letter resource type abbreviation><4 digits>

does that make reasonable sense?

**Review by karlnaden — CHANGES_REQUESTED** [review]

File: resources/dtr_bundle_patient_pat015.json [hunk 2]
```diff
@@ -2721,7 +2737,123 @@
       },
       "request": {
         "method": "PUT",
-        "url": "Task/cdex-task-example19"
+        "url": "Claim/MedicalServicesAuthorizationExample"
+      }
+    },
+    {
+      "resource": {
+        "resourceType": "Coverage",
```
[the remark this answers is not in the document]
Comment (elsaperelli) [reply resources/dtr_bundle_patient_pat015.json]: Addressed in [Remove new Coverage, change reference to cov015](https://github.com/inferno-framework/inferno-reference-server/pull/192/commits/78303f1068a5cfdf976574a5ecb20e459e368fb0)

File: resources/dtr_bundle_patient_pat015.json [hunk 3]
```diff
@@ -2721,7 +2737,123 @@
...
+      },
+      "request": {
+        "method": "PUT",
+        "url": "Organization/InsurerExample"
+      }
+    },
+    {
+      "resource": {
+        "resourceType": "Organization",
+        "id": "UMOExample",
```
[the remark this answers is not in the document]
Comment (elsaperelli) [reply resources/dtr_bundle_patient_pat015.json]: Addressed in [Change id naming conventions, align information between resources](https://github.com/inferno-framework/inferno-reference-server/pull/192/commits/d5f697bcde9e1846693ac3a4e681b7977827e912)

**Comment by karlnaden** [comment]
For resources that aren't referenced outside the bundle (E.g., in the DTR Light EHR preset), change the request method to POST so that ids assigned by the server and not fixed. HAPI should handle updating the references inside the bundle to use the newly generated id (though test and verify that!)

File: resources/dtr_bundle_patient_pat015.json [hunk 4]
```diff
@@ -2918,8 +3005,8 @@
         ]
       },
       "request": {
-        "method": "PUT",
-        "url": "AllergyIntolerance/allergyintolerance-example1"
+        "method": "POST",
+        "url": "AllergyIntolerance/allint015"
```
Comment (karlnaden) [inline resources/dtr_bundle_patient_pat015.json:3009]: Need to be careful to make sure that we're giving HAPI enough information to update references to this instance after generating a new resource id as POST tells it to do. See the [HAPI FHIR documentation](https://smilecdr.com/docs/fhir_standard/transactions.html#placeholder-ids) for how to do this. Specifically, you put the original id into the "fullUrl" element of the entry. Probably the best thing to do is anywhere you have a POST, move the `entry.request.url` value to `entry.fullUrl`.  I think that should work, but please test it!

Comment (elsaperelli) [reply resources/dtr_bundle_patient_pat015.json]: Addressed and tested in [Use fullUrls](https://github.com/inferno-framework/inferno-reference-server/pull/192/commits/a0ca0fd25376ba75415c8d73b1486e158ffd3478)

File: resources/dtr_bundle_patient_pat015.json [hunk 5]
```diff
@@ -2509,210 +2510,222 @@
...
+          ]
+        },
+        "identifier": [
           {
-            "type": {
+            "system": "http://example.org/PATIENT_EVENT_TRACE_NUMBER",
+            "value": "111099",
+            "assigner": {
+              "identifier": {
+                "system": "http://example.org/USER_ASSIGNED",
```
Comment (karlnaden) [inline resources/dtr_bundle_patient_pat015.json:2625]: in these two places, replace http://example.org with http://inferno.healthit.gov/reference-server/r4 as we've done elsewhere.

File: resources/dtr_bundle_patient_pat015.json
Comment (karlnaden) [inline resources/dtr_bundle_patient_pat015.json:1]: ran a reference resolution check and found a few places where references don't resolve:
 Resource (id):    
	path: entry[5].resource.requester, type: practitioner, id: pra-hfairchild, 
	path: entry[6].resource.requester, type: practitioner, id: pra-hfairchild, 
	path: entry[7].resource.performer[0], type: practitioner, id: pra-dmorgan, 
	path: entry[7].resource.performer[1], type: organization, id: org-lab, 
	path: entry[8].resource.performer[0], type: organization, id: org-lab, 
	path: entry[9].resource.performer[0], type: practitioner, id: pra-dmorgan, 
	path: entry[10].resource.performer[0], type: practitioner, id: pra-dmorgan, 
	path: entry[14].resource.performer[0], type: practitioner, id: pra-dmorgan, 
	path: entry[14].resource.performer[1], type: organization, id: org-lab, 
	path: entry[15].resource.performer[0], type: practitioner, id: pra-dmorgan, 
	path: entry[15].resource.performer[1], type: organization, id: org-lab, 
	path: entry[21].resource.performer[0], type: practitioner, id: pra1255

Update to practitioners and organizations in this file - I think that should be fine.

Comment (karlnaden) [inline resources/dtr_bundle_patient_pat015.json:1]: Noticed a few wonky meta.profile values for DeviceRequests.  Should be http://hl7.org/fhir/us/davinci-crd/StructureDefinition/profile-devicerequest in all three cases.

Comment (elsaperelli) [reply resources/dtr_bundle_patient_pat015.json:1]: Addressed in [Resolve references](https://github.com/inferno-framework/inferno-reference-server/pull/192/commits/eb2cf0ac2c0567246c383e815db6a3af5bda44e9)

Comment (elsaperelli) [reply resources/dtr_bundle_patient_pat015.json:1]: Addressed in [Fix meta.profile for DeviceRequest](https://github.com/inferno-framework/inferno-reference-server/pull/192/commits/ac18a54875c9642c7a03955b33385ab199b2c964)

**Review by karlnaden — APPROVED** [review]

**Pull request merged**